jstz_proto/src/operation.rs

```diff
@@ -39,6 +39,7 @@ impl Operation {
         let next_nonce = Account::nonce(rt, tx, &self.source)?;
 
         if self.nonce == *next_nonce {
+            next_nonce.increment();
```

Comment:
Just to confirm this is the correct behaviour. This will increment the nonce even if the operation later fails.